src/components/Sum.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Sum = ({ a, b }) => {
+  const aValue = a || 0;
+  const bValue = b || 0;
```

Comment:
it's alright, but in TS this approach will not work, because a and b are both required